tests/python_function/test_hashing.py

```diff
@@ -18,7 +19,7 @@
     tlibb_etl_combine_random_samples: "482acd40279d561126e281ddc57be141e3f474ae466cdfc25ab82896a71e8fba",
     tliba.summary.add_bernoulli_samples: "9f084968cc4a3baf0743f49df222ca88d32db8d241b089f6d09d1adbc9014a74",
     standalone_func: "1b4196e28cc1e2a4658d151d1a31ae77a96ac190d98d31cadc80ccbc720ef6e3",
-    wrapper_func: "e74296ec2f6c2e44970642164aba5e4804ae193ad1558750fd293f99978f53f7",
+    wrapper_func: "a6bcb86d4331a0fddb3741a583038f31b707734a2f2bf8d1297c4b358251792c",
```

Comment:
Do you know why does the hash of the func changes?

Reply:
I suspect because the FQN of the function changed

Reply:
Before, `wrapper_func` was part of `tliba` (and traced `compute_moments`), now it does not.